<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>511</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Page 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Page 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="enter">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Page1 login</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="enter_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="back_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_22">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Page 2 login</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
